zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/SessionsServiceImpl.java

```diff
@@ -124,9 +128,13 @@ public Session get(String id) throws IOException {
             // Deserialise the json:
             Session session = sessionsStore.read(sessionPath(id));
             if (!expired(session)) {
+                info().log("Session running successfully during session get");
                 updateLastAccess(session);
                 result = session;
             }
+            else{
+               warn().log("Session found expired, this is a known error during session get.");
```

Comment:
Should be lower-case and not end with a full-stop (https://github.com/ONSdigital/dp/blob/main/standards/LOGGING_STANDARDS.md#effective-event-types)

Reply:
This has now been amended